apps/src/blockly/addons/contextMenu.js

```diff
@@ -219,6 +220,38 @@ const registerDarkTheme = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(darkThemeOption);
 };
 
+/**
+ * Change workspace theme to CdoHighContrastTheme
+ */
+const registerHighContrastTheme = function() {
+  const highContrastThemeOption = {
+    displayText: function(scope) {
+      return (
+        (isCurrentTheme(HIGH_CONTRAST, scope.workspace)
+          ? '✓ '
+          : `${msg.enable()} `) + msg.blocklyHighContrastTheme()
+      );
+    },
+    preconditionFn: function(scope) {
+      if (isMusicLabTheme(scope.workspace)) {
+        return HIDDEN;
+      } else if (isCurrentTheme(HIGH_CONTRAST, scope.workspace)) {
+        return DISABLED;
+      } else {
+        return ENABLED;
+      }
+    },
+    callback: function(scope) {
+      localStorage.setItem(BLOCKLY_THEME, HIGH_CONTRAST);
+      scope.workspace.setTheme(Blockly.themes.highContrast);
+    },
+    scopeType: GoogleBlockly.ContextMenuRegistry.ScopeType.WORKSPACE,
+    id: 'highContrastTheme',
```

Comment:
Is this theme ID referenced anywhere else? Should we use a constant for it?

Reply:
It's not. This specific id is just for the context menu option element.